src/components/file-uploader/file-uploader.hbs

```diff
@@ -11,7 +11,7 @@
   <p class="{{@root.prefix}}--label-description">only .jpg and .png files. 500kb max file size.</p>
   <div class="{{@root.prefix}}--file" data-file>
     <label for="your-file-importer-id-here"
-      class="{{@root.prefix}}--file-btn {{@root.prefix}}--btn {{@root.prefix}}--btn--primary {{@root.prefix}}--btn--sm"
```

Comment:
Seems that this portion of the markup affects `v9` as well - Can we rope it off?